src/mp4box/hevc.rs

```diff
@@ -8,7 +8,7 @@ use crate::mp4box::{
 };
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize)]
-pub struct Hvc1Box {
+pub struct HevcBox {
```

Comment:
It would be nice with a docstring on this explaining if this is related to H.264 or H.265 or whatever